lib/pipeline/pipeline.js

```diff
@@ -25,8 +26,41 @@ class Pipeline {
+            </style>
+          </head>
+          <body>
+            <div class="blobs">
+              <div class="blob"></div>
+              <div class="blob"></div>
+              <div class="blob"></div>
+              <div class="blob"></div>
+            </div>
+            <svg xmlns="http://www.w3.org/2000/svg" version="1.1">
+              <defs>
+                <filter id="goo">
+                  <feGaussianBlur in="SourceGraphic" stdDeviation="10" result="blur" />
+                  <feColorMatrix in="blur" mode="matrix" values="1 0 0 0 0  0 1 0 0 0  0 0 1 0 0  0 0 0 18 -7" result="goo" />
+                  <feBlend in="SourceGraphic" in2="goo" />
+                </filter>
+              </defs>
+            </svg>
+            <h1 class="loading-msg">${__('Embark is building, please wait...')}</h1>
+          </body>
+        </html>`;
+        fs.writeFile(self.buildDir + 'index.html', html, next);
+      },
```

Comment:
Would it be possible to put that in a template file like we do here instead (https://github.com/embark-framework/embark/blob/develop/lib/contracts/code_templates/vanilla-contract.js.ejs)
Having a bunch of HTML in the script adds useless lines.